weed/s3api/s3api_bucket_handlers.go

```diff
@@ -346,7 +347,9 @@ func (s3a *S3ApiServer) GetBucketLifecycleConfigurationHandler(w http.ResponseWr
 			Expiration: Expiration{Days: days, set: true},
 		})
 	}
-
+	sort.Slice(response.Rules, func(i, j int) bool {
```

Comment:
this change seems not related.

Reply:
> this change seems not related.

@chrislusf  Yes. I checked. Recently test_lifecycle_get started fluttering due to sorting.
reverted